### PR TITLE
Nearby View: Various small tweaks

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**
The old sorting didn't work at all. But the new one is smarter and sorts by departure time! Wonderful 

Adds support for multiple operator logos per stop

**PR Checklist:**
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/viewers/nearby/stop.tsx [hunk 1]
```diff
@@ -159,18 +159,23 @@ const Stop = ({
...
       <CardHeader>
         <CardTitle>
-          <IconWithText icon={<Operator operator={operator} />}>
+          <IconWithText
+            icon={
+              <>
+                {transitOperators
+                  ?.filter((to) => Array.from(agencies).includes(to.agencyId))
+                  .map((to) => (
+                    <Operator key={to.agencyId} operator={to} />
```
Comment (amy-corson-ibigroup) [inline lib/components/viewers/nearby/stop.tsx:174]: Can we filter out duplicate operators here? 
image

File: lib/components/util/operator-logo.tsx [hunk 2]
```diff
@@ -3,6 +3,9 @@ import React from 'react'
 import styled from 'styled-components'
 
 const OperatorImg = styled.img`
+  &:not(:last-of-type) {
+    margin-right: 0.25ch;
```
Comment (amy-corson-ibigroup) [inline lib/components/util/operator-logo.tsx:7]: I think these are a little a little tight together, I wouldn't mind upping this to like .5!

**Review by amy-corson-ibigroup — CHANGES_REQUESTED** [review]
Looks good, just some notes about the multiple operator logos

File: lib/components/viewers/nearby/stop.tsx (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/components/viewers/nearby/stop.tsx:174]: How would you suggest doing that? They're different operators with different IDs, but the same image. We could filter on image url? But isn't that a bit hacky

Comment (amy-corson-ibigroup) [reply lib/components/viewers/nearby/stop.tsx:174]: Hmm in this example, even though they're different feeds, it looks like both the operators have the same `name`?

Comment (miles-grant-ibigroup) [reply lib/components/viewers/nearby/stop.tsx:174]: Good idea! Filtering on name works great

**Comment by miles-grant-ibigroup** [comment]
Great changes thanks!

**Review by amy-corson-ibigroup — APPROVED** [review]
Looks good! Thank you!

**Review by daniel-heppner-ibigroup — APPROVED** [review]
Great changes! thank you!!

**Pull request merged**